docs/audit_report/changes/topics/continuous_integration.yml

```diff
@@ -41,3 +41,20 @@ patches:
 - pr: 4078  # https://github.com/randombit/botan/pull/4078
   merge_commit: 100a065bda4219d0cea220859681e65a2714019b
   classification: info
+
+# Convert most Linux CI builds to use Ubuntu 24.04  (@randombit)
+- pr: 4098  # https://github.com/randombit/botan/pull/4098
+  merge_commit: 117f3b351a294c9c29982b11e52822faae47386d
+  classification: info
+
+# Add a workaround for MSVC/GH fuckup  (@randombit)
```

Comment:
Jack! Language! 😝

Reply:
Well, yeah. Some things demand to be called by their actual name. 😛